pkg/util/v1beta1/katibconfig/config.go

```diff
@@ -28,9 +28,10 @@ type SuggestionConfig struct {
 
 // MetricsCollectorConfig is the JSON metrics collector structure in Katib config.
 type MetricsCollectorConfig struct {
-	Image           string                      `json:"image"`
-	ImagePullPolicy corev1.PullPolicy           `json:"imagePullPolicy"`
-	Resource        corev1.ResourceRequirements `json:"resources"`
+	Image            string                      `json:"image"`
+	ImagePullPolicy  corev1.PullPolicy           `json:"imagePullPolicy,omitempty"`
+	Resource         corev1.ResourceRequirements `json:"resources,omitempty"`
+	WaitAllProcesses *bool                       `json:"waitAllProcesses,omitempty"`
```

Comment:
Can you add `omitempty` to the `SuggestionConfig` also please ? It should be in all parameters but `Image`.